src/ProjectOrigin.Registry.Server/Services/TransactionProcessor.cs

```diff
@@ -67,7 +62,8 @@ private async Task ProcessTransaction(V1.Transaction transaction)
             var verifiableEvent = new VerifiableEvent(eventId, transactionHash, transaction.ToByteArray());
             await _eventStore.Store(verifiableEvent);
 
-            _logger.LogInformation($"Completed transaction {transactionHash}");
+            await context.NotifyCompleted();
```

Comment:
Is this explicit call to `NotifyCompleted` needed for the JobConsumer to complete?

Reply:
I chose to add it there, so it happens before the end of try-catch statement. Everything after this is errorhandling.